app/src/main/java/org/mozilla/focus/ui/dialog/FocusDialog.kt

```diff
@@ -139,21 +149,37 @@ fun DialogTextButton(
 fun DialogInputField(
     modifier: Modifier = Modifier,
     text: String,
+    placeholder: String,
     onValueChange: (String) -> Unit
 ) {
-    TextField(
-        modifier = modifier
-            .wrapContentHeight(),
-        value = text,
-        onValueChange = onValueChange,
-        textStyle = MaterialTheme.typography.h1,
-        colors = TextFieldDefaults.textFieldColors(
-            backgroundColor = focusColors.secondary,
-            textColor = focusColors.onSecondary
-        ),
-        singleLine = true,
-        shape = MaterialTheme.shapes.large
-    )
+    Column {
+        BasicTextField(
+            modifier = modifier
+                .wrapContentHeight()
+                .fillMaxWidth(),
+            value = text,
+            onValueChange = onValueChange,
+            textStyle = focusTypography.dialogInput,
+            decorationBox = { innerTextField ->
+                if (text.isEmpty()) {
+                    Text(
+                        text = placeholder,
```

Comment:
Instead of using this workaround, we can investigate using a `TextField` composable that has a `placeholder` parameter that also comes with a customisable default style.

Reply:
It seems that as `TextField` is a Material design implementation of a text field, they have chosen to hardcode some style-related values, like padding for example: `internal val TextFieldPadding = 16.dp.
This means that using `TextField` will give us no option to adjust the start padding for the input field.

I still feel we should go with the Material design implementation that includes the padding because it comes with out-of-the-box benefits like placeholder handling and optional icons. (And as the name suggests, it follows Material design guidelines)

As we do not have a design for the rename dialog, I think @jeffreygee will need to help with some input regarding the padding and general design of the dialog.
`

Reply:
This is the dialog with the mentioned padding, as a reference:
with text / with the placeholder
<img src = https://user-images.githubusercontent.com/48995920/145412548-e57453dd-a189-4ba3-9b6e-3c86416a4417.png width = 35%> <img src =https://user-images.githubusercontent.com/48995920/145412824-92ea6503-fc1e-4e89-b71c-dcd76364d310.png width = 35%>